rnd/autogpt_builder/src/app/page.tsx

```diff
@@ -32,7 +34,7 @@ export default function Home() {
 
           <div className="w-full flex justify-center mt-10">
               <div className="flow-container w-full h-full">
-                  <Flow/>
+                  <FlowEditor flowID={useSearchParams().get("flowID") ?? undefined} />
```

Comment:
Can we be more mindful if you want to rename - this will cause conflicts in the current WIPs - do you think this is 100% necessary for the functionality in this PR or can the rename be introduced in a different PR?

Reply:
I have to rename *something*. Otherwise the `Flow` type and `Flow` component clash. I thought it made more sense to rename `<Flow>` to better represent its functionality.

Reply:
I was thinking the type should be called graph since it's an API representation. But i don't think it's a blocker, just an idea

Reply:
I guess we could do that. But that would also require a refactor of the API client for consistency. Could do later in a PR if that's something we want.